[IMP] html_editor: improve wysiwyg button hover effect

ldau-odoo opened the pull request:
This PR improves the WYSIWYG button hover effect to better highlight the intended element.

The mentioned task is focused on the AI button; it was a good opportunity to improve the hover on the other buttons as well.

task-5093189

PR-enterprise: https://github.com/odoo/enterprise/pull/94944



---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/227632.png)](https://mergebot.odoo.com/odoo/odoo/pull/227632)

Brieuc-brd reviewed (approved):
LGTM, thanks 👍

ldau-odoo commented:
Hello @dmo-odoo,

The word choice wasn’t correct; I’ve updated the commit message to make it clearer.
The bug you mentioned was due to the linked PR; thanks, I’ve fixed it!
